portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 9.082, lng: 8.675}, zoom: 5});
```

Comment:
nit: Ideally you should define these as constant variables in some central place.